solutions/swb-reference/integration-tests/support/complex/accountHelper.ts

```diff
@@ -0,0 +1,153 @@
+import { PolicyDocument } from '@aws-cdk/aws-iam';
+import { GetBucketPolicyCommandOutput, PutBucketPolicyCommandInput } from '@aws-sdk/client-s3-control';
+import { AwsService } from '@aws/workbench-core-base';
+import { IamHelper } from '@aws/workbench-core-datasets';
+import _ from 'lodash';
+import Setup from '../setup';
+import Settings from '../utils/settings';
+
+export class AccountHelper {
+  private _awsSdk: AwsService;
+  private _settings: Settings;
+  public constructor(awsSdkClient: AwsService) {
+    const setup: Setup = new Setup();
+    const settings: Settings = setup.getSettings();
+    this._settings = settings;
+    this._awsSdk = awsSdkClient;
+  }
+
+  public async listOnboardedAccounts(): Promise<{[id: string]: string}[]> {
+    const queryParams = {
```

Comment:
This should probably just be an API call to list accounts. If we don't have that API yet, can we add a TODO to this code segment to replace this with just calling the API.